pages/country.js

```diff
@@ -45,8 +46,19 @@ const AnimatedHeading = styled(Heading)`
 `
 
 export default class Country extends React.Component {
-  static async getInitialProps ({ query }) {
+  static async getInitialProps ({ req, res, query } ) {
     const { countryCode } = query
+
+    if (res && (countryCode !== countryCode.toUpperCase())) {
+      res.writeHead(301, {
+        Location: `/country/${countryCode.toUpperCase()}`
+      });
+
+      res.end();
+      return {};
```

Comment:
Remove the semicolons. We don't use them in our coding style.

Reply:
Removed